grails-app/migrations/changelog.groovy

```diff
@@ -6,8 +6,4 @@ databaseChangeLog = {
     //  - Add a new directory (e.g. 0.10.x) for new changelogs
     //  - Replace this changelog with the new version (e.g. 0.10.x/changelog.xml)
     include file: '0.9.x/changelog.xml'
-
-    // Recreate all views AFTER all changesets have been applied (just in case)
-    include file: 'views/changelog.xml'
-
```

Comment:
I hoisted this to BootStrap.groovy, since it destroys views before doing any migrations, it seemed more clear to have it build them back after doing all migrations.

Reply:
I don't want to run migrations programmatically if we don't have to. I would prefer if the drop / create views were done in the changelog.xml.

Reply:
This isn't resolved, but we can revisit this later so we can get this pushed to obnavtest1.